draft-packages/form/KaizenDraft/Form/Primitives/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -59,11 +51,7 @@ const ToggleSwitch: ToggleSwitch = ({
         onBlur={onBlur}
         disabled={disabled}
       />
-      <span
-        className={classnames(styles.track, {
-          [styles.freemium]: theme === ToggleTheme.FREEMIUM,
```

Comment:
The sass for this class should be removed now as well. One of the disadvantages of our CSS Modules set up is that there's no linting to check if classes that exist in scss files are actually being used in TS, and vice versa. As a result, we have to make sure we manually catch cases like this where we're going to end up with dead code.

Reply:
@dougmacknz nice spot - I just did another check through the `styles.scss` and there seems to be no references of the `.freemium` classname

Reply:
Wow, so the freemium theme was doing nothing 😑